ui/src/app/user-info/components/user-info-overview/user-info-overview.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+
+import {DataLoader, Page} from '../../../shared/components';
+import {services} from '../../../shared/services';
+
+require('./user-info-overview.scss');
+
+export const UserInfoOverview = () => (
+    <Page title='User Info' toolbar={{breadcrumbs: [{title: 'User Info'}]}}>
+        <div className='user-info'>
+            <div className='argo-container'>
+                <div className='user-info-overview__panel white-box'>
+                    <DataLoader key='userInfo' load={() => services.users.get()}>{(userInfo) => (
+                        userInfo.loggedIn ? (
+                            <React.Fragment key='userInfoInner'>
+                                <p key='username'>Username: {userInfo.username}</p>
```

Comment:
we display: 

* Username
* Issuer
* Groups

Do we need anything else (from the JWT payload)?